e2e_playwright/multipage_apps_v2/mpa_v2_basics_test.py

```diff
@@ -240,15 +240,15 @@ def test_removes_query_params_with_st_switch_page(page: Page, app_port: int):
     expect(page).to_have_url(f"http://localhost:{app_port}/page_5")
 
 
-def test_removes_query_params_when_clicking_link(page: Page, app_port: int):
+def test_removes_query_params_when_clicking_link(app: Page, app_port: int):
     """Test that query params are removed when swapping pages by clicking on a link"""
 
-    page.goto(f"http://localhost:{app_port}/page_7?foo=bar")
-    wait_for_app_loaded(page)
+    app.goto(f"http://localhost:{app_port}/page_7?foo=bar")
+    wait_for_app_loaded(app)
 
```

Comment:
nit: should there be an expect for `page_7` here like `expect(app).to_have_url(f"http://localhost:{app_port}/page_7")` just to ensure that the first `goto` worked properly? it might help in case of an issue to narrow down whether its a general app think (then this expect would fail) or rather a page_link issue, then the `page_4` link expectation would fail